source/models/mock_data.json

```diff
@@ -41,7 +41,11 @@
           },
           "events": [
             {
+<<<<<<< HEAD
+              "description": "June 3, 2021",
+=======
               "description": "June 10, 2021",
+>>>>>>> 61b572caf579b3f16a261b99ce79cd3095a3157a
```

Comment:
Before we merge this PR, let's resolve this conflict

Reply:
It doesn't really matter which one we store right? the time stamp is consistent

Reply:
Resolved, the timestamp is actually for June 10, 2021 so to prevent confusion we should have the description be same.